lib/matplotlib/backends/backend_gtk.py

```diff
@@ -1003,13 +999,13 @@ def on_combobox_lineprops_changed(self, item):
         if marker is None: marker = 'None'
         self.cbox_markers.set_active(self.markerd[marker])
 
-        r,g,b = colorConverter.to_rgb(line.get_color())
-        color = gtk.gdk.Color(*[int(val*65535) for val in (r,g,b)])
+        rgba = mcolors.to_rgba(line.get_color())
```

Comment:
OK, I have restored `to_rgb`, but will wait until we get a response re. GTK to decide what to do here (and won't push for now).